src/components/TopbarViewer/TopbarViewer.style.tsx

```diff
@@ -35,22 +36,26 @@ export const SearchbarContainer = styled.div`
   }
 `
 
-export const StyledSearchbar = styled(Searchbar)<FocusProps>`
-  transition: width ${transitions.timings.regular} ${transitions.easing};
-  will-change: width;
-  width: ${({ hasFocus }) => (hasFocus ? '100%' : '39px')};
-  padding-left: ${({ hasFocus }) => (hasFocus ? sizes(4) : 0)};
-  margin-left: auto;
-  height: 39px;
+export const ButtonWrapper = styled.div`
+  align-self: center;
+  justify-self: flex-end;
+  flex-shrink: 0;
+`
 
-  ${media.md} {
-    max-width: 480px;
-    width: 100%;
-    margin-left: 0;
-    height: initial;
+export const Overlay = styled.div`
```

Comment:
I fixed it but I'm not sure what do you mean by 'navigating using TAB'. Navigating using keyboard is not part of this PR, it will be accomplished in #1427

Reply:
Oh, I see, didn't check fully all the designs, sorry. I still don't like this. Other video websites like vimeo, youtube, or twitch doing this differently. I will talk to @toiletgranny on Monday about this and add a comment in the #1473 as well. 
We can resolve this conversation.